LanguageExt.Core/HKT/OptionT.cs

```diff
@@ -12,73 +12,73 @@ namespace LanguageExt
     /// <typeparam name="OuterType"></typeparam>
```

Comment:
This whole file can be deleted.  It was an early experiment, not needed any more.